service/src/test/java/bio/terra/cbas/dependencies/wds/TestScheduledWdsClientRefresher.java

```diff
@@ -0,0 +1,88 @@
+package bio.terra.cbas.dependencies.wds;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.ws.rs.client.Client;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class TestScheduledWdsClientRefresher {
+
+  @Test
+  void canGetCurrentClient() {
+    ScheduledWdsClientRefresher scheduledWdsClientRefresher = new ScheduledWdsClientRefresher();
+    assert (scheduledWdsClientRefresher.getCurrentClient() != null);
```

Comment:
Unless you run with assertions enabled (and most terra code doesn't do this), `assert` doesn't do anything. See https://docs.oracle.com/javase/8/docs/technotes/guides/language/assert.html#enable-disable. You need to use `assertTrue()` or (my preference) a hamcrest matcher
```suggestion
    assertThat(scheduledWdsClientRefresher.getCurrentClient(), nullValue());
```

Reply:
🤦 ... and even better, `assertNotNull`